src/map.rs

```diff
@@ -575,6 +659,99 @@ impl MapData {
             view_box,
         )))
     }
+
+    fn parse_map_info(&mut self, info: MapV2Info) {
+        let parse_coords = |s: &str| -> Option<(f32, f32)> {
+            let mut it = s.splitn(2, ',');
+            let x = it.next()?.parse::<f32>().ok()?;
+            let y = it.next()?.parse::<f32>().ok()?;
+            Some((x, y))
+        };
+
+        let mut outlines = Vec::new();
+        let mut areas = Vec::new();
+        let mut block_lines = Vec::new();
+
+        for group in &info {
+            let Some(first) = group.first() else { continue };
+            match first.as_str() {
+                "1" => {
```

Comment:
Please create constant for it with a name that describes it

Reply:
Done